production/helm/README.md

```diff
@@ -56,3 +56,21 @@ $ kubectl port-forward --namespace <YOUR-NAMESPACE> service/loki-grafana 3000:80
 
 Navigate to http://localhost:3000 and login with `admin` and the password output above.
 Then follow the [instructions for adding the loki datasource](/docs/usage.md), using the URL `http://loki:3100/`.
+
+## How to contribute
+
+If you want to add any feature to helm chart, you can follow as below:
+
```

Comment:
I would also recommend to lint all chart for errors or create a lint make target and document its usage here.

Reply:
@Kuqd The `make helm` already does linting.

Reply:
Should we ask for bumping version also ? May be good to give information about this requirement too.

Reply:
Sure, I also ask to bump chart in [PR template](https://github.com/grafana/loki/pull/517). Most people especially don't understand that they need to bump loki-stack chart as well. Once Loki has a release, we can remove this restriction since chart versioning will be handled by release process.

Reply:
@daixiang0 please ^

Reply:
so i do not need to change it, right?